labelbox/schema/batch.py

```diff
@@ -23,3 +32,45 @@ class Batch(DbObject):
     # Relationships
     project = Relationship.ToOne("Project")
     created_by = Relationship.ToOne("User")
+
+    def export_data_rows(self, timeout_seconds=120) -> Generator:
```

Comment:
is this the same output structure as dataset?

Reply:
yes